Source/JavaScriptCore/bytecode/BytecodeList.rb

```diff
@@ -1949,6 +1949,7 @@
         arrayref: VirtualRegister,
         index: VirtualRegister,
         typeIndex: unsigned,
+        arrayGetKind: unsigned,
```

Comment:
The opcodes were collapsed, so `array_get` gets an extra argument for which kind of `get` to run.